integrations/jira/client.go

```diff
@@ -0,0 +1,83 @@
+package jira
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkintegrations"
+	"go.autokitteh.dev/autokitteh/sdk/sdkmodule"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+type integration struct {
+	vars sdkservices.Vars
+}
+
+var integrationID = sdktypes.NewIntegrationIDFromName("jira")
+
+var desc = kittehs.Must1(sdktypes.StrictIntegrationFromProto(&sdktypes.IntegrationPB{
+	IntegrationId: integrationID.String(),
+	UniqueName:    "jira",
+	DisplayName:   "Atlassian Jira",
+	Description:   "Atlassian Jira is an issue tracking and project management system.",
+	LogoUrl:       "/static/images/jira.svg",
+	UserLinks: map[string]string{
+		"1 REST API":          "https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/",
+		"2 Go client API":     "https://pkg.go.dev/github.com/andygrunwald/go-jira",
+		"3 Python client API": "https://jira.readthedocs.io/",
+	},
+	ConnectionUrl: "/jira/connect",
+	ConnectionCapabilities: &sdktypes.ConnectionCapabilitiesPB{
+		RequiresConnectionInit: true,
+	},
+}))
+
+func New(cvars sdkservices.Vars) sdkservices.Integration {
+	i := &integration{vars: cvars}
+	return sdkintegrations.NewIntegration(
+		desc,
+		sdkmodule.New( /* No exported functions for Starlark */ ),
+		connStatus(i),
+		connTest(i),
+	)
+}
+
+func connStatus(i *integration) sdkintegrations.OptFn {
```

Comment:
you can just do `func (i *integrtation)`, and supply it as a parameter above to NewIntegration, I think it's nicer.

Reply:
-0